cider/exec/plan/parser/ConverterHelper.cpp

```diff
@@ -43,18 +43,18 @@ void registerExtensionFunctions() {
 std::unordered_map<int, std::string> getFunctionMap(const substrait::Plan& plan) {
   std::unordered_map<int, std::string> function_map;
   for (int i = 0; i < plan.extensions_size(); i++) {
-    auto extension = plan.extensions(i);
+    const auto& extension = plan.extensions(i);
     if (extension.has_extension_function()) {
-      auto function = extension.extension_function().name();
-      auto function_name = function.substr(0, function.find_first_of(':'));
-      function_map.emplace(extension.extension_function().function_anchor(),
-                           function_name);
+      const auto& function = extension.extension_function().name();
+      // do function lookup verify and function mapping
+      // get op type, no need mapping in substraitToAnalyzerExpr
+      function_map.emplace(extension.extension_function().function_anchor(), function);
     }
   }
-  return function_map;
+  return std::move(function_map);
 }
 
-std::string getFunctionName(const std::unordered_map<int, std::string> function_map,
+std::string getFunctionName(const std::unordered_map<int, std::string>& function_map,
```

Comment:
So here we get not only the function name? if so, rename it to `getFunction`?

Reply:
Done. Changed it to getFunctionSignature, which the signature string is used for function lookup.